R/oauth-flow-auth-code.R

```diff
@@ -403,23 +403,10 @@ oauth_flow_auth_code_pkce <- function() {
   )
 }
 
-# Try to determine whether we can redirect the user's browser to a server on
-# localhost, which isn't possible if we are running on a hosted platform.
-#
-# Currently this detects RStudio Server, Posit Workbench, and Google Colab. It
-# is based on the strategy pioneered by the {gargle} package.
-is_hosted_session <- function() {
```

Comment:
moved to `utils.R` to:
1. Live with `is_rstudio_session()` function (new helper for `prompt_user()`)
2. Reflect usage beyond only `oauth-flow-auth-code` (#413)

Reply:
Perfect, thanks.